src/serverModuleGen.py

```diff
@@ -5,9 +5,12 @@ def getModules(filename):
         modules = configfile.readlines()
         ret = []
         for module in modules:
-            module = (module.rstrip()).lstrip()
+            module = module.strip()
             if len(module) > 0 and module[0] != '#':
-                ret.append(module)
+                if module[0] == '/':
+                    ret.append(module[module.rfind('/')+1:])
+                else:
+                    ret.append(module)
```

Comment:
This works so no change requested, but if you wanted to tighten it up a bit
```python
    for module in modules:
        module = module.split("#")[0].split("/")[-1].strip()
        if module:
            ret.append(module)
```
This also handles trailing comments i.e. `MyMsg  # This is my fancy module` -> `MyMsg`

If you want to go full blown crazy the whole func in 3 lines:
```python
def getModules(filename):
    with open(filename) as configfile:
        return [m.split("/")[-1].strip() for m in configfile.readlines() if m.split("#")[0].strip()]
```
but that is a bit unreadable.

Reply:
Ooh, I like the fancy Python syntax, switched to the first option, thanks!